operator-pipeline-images/Dockerfile

```diff
@@ -61,6 +61,18 @@ WORKDIR /home/user
 
 COPY ./operator-pipeline-images ./
 
+# install PDM
+RUN pip3 install --upgrade pip setuptools wheel && pip3 install pdm
+# install dependencies in virtual environment
+COPY ./pdm.lock ./pyproject.toml ./
+RUN pdm venv create 3.11 && pdm install --no-lock --no-editable \
```

Comment:
We should keep the Python version consistent, so this should be 3.10 - as used by latest F36. Also in other places.